tests/test_printer.py

```diff
@@ -399,10 +398,41 @@ def test_print_block_without_arguments():
     assert io.getvalue() == """%0 : i32, %1 : i32\n  "test.op"(%1) : (i32) -> ()"""
 
 
+def test_print_block_with_terminator():
+    """Print a block and with its terminator."""
+    block = Block(ops=[test.TestOp.create(), test.TestTermOp.create()])
+
+    # Print block ops including block terminator
+    io = StringIO()
+    p = Printer(stream=io)
+    p.print_block(block, print_block_terminator=True)
+    assert (
+        io.getvalue()
+        == """\n^0:\n  "test.op"() : () -> ()\n  "test.termop"() : () -> ()"""
```

Comment:
```suggestion
        == """
        ^0:
          "test.op"() : () -> ()
          "test.termop"() : () -> ()
        """
```
nit but feels like it's a little more readable with the inline newlines